app/assets/stylesheets/new_board/_story.scss

```diff
@@ -341,11 +341,21 @@
         &__controls {
           @extend .btn-group;
 
+          height: 22px;
```

Comment:
Are you sure that you need to define height here too?

Reply:
Yes, without height the div gets size 0